molecule/mtls-customcerts-remote-src-rhel/molecule.yml

```diff
@@ -0,0 +1,131 @@
+---
+### Installation of Confluent Platform on CentOS7.
+### MTLS enabled with custom certificates on remote host
+
+driver:
```

Comment:
Let's once discuss in DSM if we can add a test for this in any of our existing molecule scenario.

Reply:
Have incorporated your suggestions and updated the PR, please review again.